fuels-abigen-macro/tests/test_projects/call_empty_return/Forc.lock

```diff
@@ -0,0 +1,21 @@
+[[package]]
+name = 'contract_test'
+dependencies = [
+    'core git+https://github.com/FuelLabs/sway-lib-core?reference=master#c331ed20ebc9d646acec6b8ee8f408627ce3b573',
+    'std git+https://github.com/FuelLabs/sway-lib-std?reference=master#4eb73be7be90e40099ba9fe34cd2d5f1faa01c6f',
+]
+
+[[package]]
+name = 'core'
+source = 'git+http://github.com/FuelLabs/sway-lib-core?reference=master#c331ed20ebc9d646acec6b8ee8f408627ce3b573'
+dependencies = []
+
+[[package]]
+name = 'core'
+source = 'git+https://github.com/FuelLabs/sway-lib-core?reference=master#c331ed20ebc9d646acec6b8ee8f408627ce3b573'
+dependencies = []
+
+[[package]]
+name = 'std'
+source = 'git+https://github.com/FuelLabs/sway-lib-std?reference=master#4eb73be7be90e40099ba9fe34cd2d5f1faa01c6f'
+dependencies = ['core git+http://github.com/FuelLabs/sway-lib-core?reference=master#c331ed20ebc9d646acec6b8ee8f408627ce3b573']
```

Comment:
I just noticed that `core` is getting pinned multiple times due to the difference in `http` (via `std` lib's dependency) and `https` URLs (via `contract_test`'s dependency). I think `https` as in the `contract_test` manifest is the better convention and we should consider updating `std`'s manifest to do the same, however this is something that `forc` should resolve either way. I've opened https://github.com/FuelLabs/sway/issues/931 to track this, but it shouldn't cause any issues in the meantime and I think it's fine to land this as is.